news/2020-11-26-community-mentorship-program-winter-2021-signups-now-open.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Community Mentorship Program Winter 2021 Signups Now Open
+date: 2020-11-26 08:00:00 +0000
+---
+
+With holiday cheer and an upcoming new year, the 17th cycle of the Community Mentorship program is on its way! If you've ever found yourself wanting to expand your current knowledge and skills or help beginner mappers get started, this program will serve as a great opportunity for you!
+
+![](/wiki/shared/news/banners/community-mentorship-program.jpg)
+
+The Community Mentorship program serves as a place for upcoming mappers and modders to develop their skills under the guidance of more experienced mappers and modders in the community. 
```

Comment:
```suggestion
The Community Mentorship program serves as a place for upcoming creators to develop their skills under the guidance of more experienced mappers and modders from the community. 
```

Reply:
Generalizing like that makes things unnecessarily broad imo. "creator" can be interpreted beyond mapper/modder eg. storyboarder or skinner, which we don't cater to.